helix-core/src/main/java/org/apache/helix/task/TaskDriver.java

```diff
@@ -645,6 +645,10 @@ public void delete(String workflow) {
    * @param forceDelete, CAUTION: if set true, workflow and all of its jobs' related zk nodes will
    *          be clean up immediately from zookeeper, no matter whether there are jobs
    *          are running or not.
+   *
+   *          Enabling this option can cause a ZooKeeper delete failure as Helix might
```

Comment:
Also could you please add that it may corrupt the Task Framework cache. At any rate, it shouldn't be used while the controller is running or the cluster is active.